kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml

```diff
@@ -89,12 +89,14 @@
    </p>
             <div xml:id="kzz7yh-f24075-Dd1e149">
                <head xml:id="kzz7yh-f24075-Hd1e151">Quaestio 1</head>
+               <head type="question-title">Utrum demones sunt compositi ex spirituali et corporali substantia</head>
+               <head type="question-title">Change this</head>
                <p xml:id="kzz7yh-f24075-d1e153">
       ¶ Quaestio I. 
       <lb ed="#V" n="7"/>PRimo ostendo quod demones sunt 
       <lb ed="#V" n="8"/>compositi ex 
       spiritua<lb ed="#V" n="9" break="no"/>li et corporali substantia. Aug. 8. de ci. cap. 15. dicit 
-      <lb ed="#V" n="10"/>quod demones sunt aerei. Et. xi. lib. 22. cap. quod 
+      <lb ed="#V" n="10"/>quod demones sunt aerei<!--word?-->. Et. xi. lib. 22. cap. quod 
```

Comment:
```suggestion
      <lb ed="#V" n="10"/>quod demones sunt aerei. Et. xi. lib. 22. cap. quod 
```

Reply:
Yes, "aereus, -a, -um," the adjective meaning "bronze," from "aes, aeris," f., "bronze."